src/components/AccountDrawer/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -87,81 +78,114 @@ const L2LogoContainer = styled.div<{ $backgroundColor?: string }>`
   justify-content: center;
 `
 
-/**
- * Renders an image by prioritizing a list of sources, and then eventually a fallback triangle alert
- */
-export function PortfolioLogo({
-  chainId = ChainId.MAINNET,
-  accountAddress,
-  currencies,
-  images,
-  size = '40px',
-  style,
-}: MultiLogoProps) {
-  const chainInfo = getChainInfo(chainId)
-  const squareLogoUrl = chainInfo?.squareLogoUrl
-  const logoUrl = chainInfo?.logoUrl
-  const chainLogo = squareLogoUrl ?? logoUrl
+type DoubleLogoProps = {
```

Comment:
nit: is there a reason these are all `types` and not `interfaces` - I think the codestyle are for these to be `interface`